test-runner-frontend/src/Views/Dashboard.js

```diff
@@ -0,0 +1,550 @@
+      } else {
+        setTestRunResults(allRows.filter(test => test.machine === machineSelection));
+      }
+    }
+  }
+
+  const handleChangeTestRunSelection = (event) => {
+    setTestRunSelection(event.target.value);
+  };
+
+  const handleChangeMachineSelection = (event) => {
+    setMachineSelection(event.target.value);
+  };
+
+
+  const sortDate = () => {
+    allRows?.sort((a, b) => b.date - a.date);
+    setTestRunResults(allRows);
+  }
+
+  const [checkBoxSelection, setCheckBoxSelection] = React.useState([]);
+
+  const updateSelectedTestRunsToCompare = (childToParentData) => {
+    if (checkBoxSelection.find(checkBoxSelection => checkBoxSelection === childToParentData))
+      setCheckBoxSelection(checkBoxSelection.filter(checkBoxSelection => checkBoxSelection !== childToParentData))
+    else
+      setCheckBoxSelection(checkBoxSelection => [...checkBoxSelection, childToParentData]);
+  }
+
+
+  useEffect(() => {
+    drowpdownMenusManager();
+  }, [testRunSelection]);
+
+
+  useEffect(() => {
+    drowpdownMenusManager();
+  }, [machineSelection]);
+
+
+  return (
+    <div>
+
+      <img
+        style={{
+          marginLeft: "2%",
+          marginTop: "0.5%",
+          marginBottom: "0.7%",
+          width: "18%"
+        }}
+        src={GraphQL_Logo}
+        alt="GraphQL Java Logo"
+      />
+
+      <Typography sx={{ marginLeft: "2%", marginBottom: "0.4%" }} variant="h4">
+        <b>Summary</b>
+      </Typography>
+      <br></br>
+
+
+      <Stack sx={{ marginBottom: "2%" }} direction="row" spacing={9}>
+
+        <Box
+          sx={{
+            width: "100%",
+            marginLeft: "2%",
+            marginRight: "2%",
+          }}>
+          <Stack direction="row">
+            <Typography variant="h5">
+              <b>Test Run</b>
+            </Typography>
+            {
+              cancelButtonState
+                ?
+                <Button sx={{
+                  color: "gray",
+                  borderColor: "gray",
+                  borderRadius: "9%",
+                  position: "absolute",
+                  right: "10.2%",
+                  size: "small"
+                }}
+                  variant="outlined"
+                  onClick={() => handleCancel()}
+                >
+                  Cancel
+                </Button>
+                :
+                null
+            }
+            <Button sx={{
+              color: !isCheckBoxActive ? "gray" : "#e535ab",
+              borderColor: !isCheckBoxActive ? "gray" : "#e535ab",
+              borderWidth: "2px",
+              borderRadius: "9%",
+              position: "absolute",
+              right: "2%",
+              size: "small"
+            }}
+              variant="outlined"
+              disabled={isCheckBoxActive && checkBoxSelection.length !== 2 ? true : false}
+              onClick={() => manageCompareAction()}
+            >
+              Compare
+            </Button>
+          </Stack>
```

Comment:
as we've not fully implemented this yet and will be later ticket, can we please comment this block. also i know ive just told you to remove all commented out code but we'll want to likely keep it for when we do that ticket. unless you want to remove it for the time being.

Reply:
Yes, I commented that block as well as code that will be used for next ticket in order to get rid of the lint warnings.